decidim-admin/app/controllers/decidim/admin/application_controller.rb

```diff
@@ -7,7 +7,9 @@ class ApplicationController < ActionController::Base
       include NeedsAuthorization
       include FormFactory
       include LocaleSwitcher
+      helper Decidim::ReplaceButtonsHelper
       helper Decidim::DecidimFormHelper
+      helper Decidim::ReplaceButtonsHelper
```

Comment:
<img width="366" alt="screen shot 2017-02-20 at 09 42 26" src="https://cloud.githubusercontent.com/assets/491891/23117634/f55b973c-f750-11e6-8c38-052a7ee06008.png">

You're adding it twice